packages/devtools-network-console/src/ui/RequestBody/index.tsx

```diff
@@ -196,6 +202,7 @@ export function RequestBody(props: IRequestBodyEditorProps) {
                         updateRowFileInfo={(id, type, fileName, contents) => {
                             dispatch(editBodyFormDataFileParams(props.requestId, id, fileName, contents, type));
                         }}
+                        fallbackFocusTargetSelector="#bodyFD"
```

Comment:
Nit: should we consider putting these in a variable somewhere in case the underlying id's change? Might not make sense if this is the only place it's used though. Up to you

Reply:
The underlying ID is contained entirely in this file, so I didn't think it was worth doing this here.